crates/red_knot_python_semantic/resources/mdtest/loops.md

```diff
@@ -0,0 +1,241 @@
+# Loops
```

Comment:
Similar to some above cases, let's instead have a `loops/` directory with e.g. `for_loop.md`, `while_loop.md` files in it.